### PR TITLE
Swap of optimizer.step and scheduler.step for lm finetuning examples

### PR DESCRIPTION
**Pull request by Morizeyao**
Prior to PyTorch 1.1.0, the learning rate scheduler was expected to be called before the optimizer’s update; 1.1.0 changed this behavior in a BC-breaking way. If you use the learning rate scheduler (calling scheduler.step()) before the optimizer’s update (calling optimizer.step()), this will skip the first value of the learning rate schedule. If you are unable to reproduce results after upgrading to PyTorch 1.1.0, please check if you are calling scheduler.step() at the wrong time.

This is my first time very simple PR, please correct me if there's anything done wrong xD.

[link](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate)

### CODE REVIEW

**Comment by thomwolf** [comment]
Looks good to me, thanks @Morizeyao!
